tt_metal/hw/ckernels/wormhole_b0/metal/llk_api/llk_sfpu/ckernel_sfpu_rand_uint.h

```diff
@@ -0,0 +1,32 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
```

Comment:
Can this also be added for Blackhole so we have same set of operations for both devices?

Reply:
Frankly, I don't have access to the Blackhole device. I can't test if this would also work on Blackhole.

Reply:
ok, can you make an issue that is missing impl for BH after you merge this PR? And make sure to add the "blackhole" tag, and to also add the project "Blackhole"

Reply:
Ok, got it.